File: src/Qweree.QAdmin.WebApplication/src/app/model/authentication/TokenInfo.ts
export class TokenInfo {

  constructor(
    public accessToken: string,
    public refreshToken: string,
    public expiresAt: number,
    public createdAt: string
  ) { }

}

export class UserInfo {
  constructor(
    public userId: string,
    public username: string,
    public fullName: string,
    public email: string,
    public role: string[],
  ) {
  }
}

File: src/Qweree.Authentication.Sdk/Tokens/TokenInfo.cs
using System;

namespace Qweree.Authentication.Sdk.Tokens
{
    public class TokenInfo
    {
        public TokenInfo(string accessToken, DateTime expiresAt) : this(accessToken, null, expiresAt)
        {
        }

        public TokenInfo(string accessToken, string? refreshToken, DateTime expiresAt)
        {
            AccessToken = accessToken;
            RefreshToken = refreshToken;
            ExpiresAt = expiresAt;
        }

        public string AccessToken { get; }
        public string? RefreshToken { get; }
        public DateTime ExpiresAt { get; }
    }
}